### PR TITLE
Dashboard: add styles for media, links to devices, titles to blocks

### PR DESCRIPTION
**Pull request by RostyslavSt**

### CODE REVIEW

**Review by LanaAvry — APPROVED** [review]
Good

File: src/pages/Dashboard/Dashboard.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   margin-bottom: 20px;
 }
 
-@media screen and (max-width: 800px) {
+@media screen and (max-width: 850px) {
```
Comment (totoshk) [inline src/pages/Dashboard/Dashboard.scss:7]: Why did you use these media instead of those specified in the variables??? Look at file "_variables.scss".

**Review by totoshk — COMMENTED** [review]
Look my comment in code.

**Review by Kaderimon — APPROVED** [review]
looks good enough, exept variables in media))

**Review by AnastasiiaHamaiunova — APPROVED** [review]
I agree, but in ValueStyle.scss, please, change color, background, etc, on color in variables, or add your own variables in _variables.scss file.

**Pull request merged**